PocketKit/Sources/PocketKit/Home/HomeCarouselItemCell.swift

```diff
@@ -165,8 +164,12 @@ extension HomeCarouselItemCell {
 
         if model.attributedTimeToRead.string.isEmpty {
             timeToReadLabel.isHidden = true
+            subtitleStack.setCustomSpacing(0, after: timeToReadLabel)
+            subtitleStack.setCustomSpacing(16.0, after: domainLabel)
```

Comment:
best solution in UIKit I can find for adding padding to the bottom of a label in a vertical stack